sponsor-dapp-v2/src/components/Step2.js

```diff
@@ -34,6 +34,7 @@ function Step2(props) {
       value: moment.unix(expiry).format("MMMM DD, YYYY LTS")
     };
   });
+  timeline.push({ key: "0", value: "Perpetual" });
```

Comment:
Just curious, why did you decide to add perpetual to all instruments in the UI rather than just adding it to the list of expiries in the config so it can be disabled for some and enabled for others?

Reply:
Yeah we could totally do that too. Should we enable it for all current products to start with?

Reply:
Yeah, I think for right now we want all of them to have a perpetual option. The only reason I think this is slightly better is that we may ultimately want to disable it for a price feed that has a true, embedded expiry date (like a specific futures contract, for instance).

Reply:
Done.